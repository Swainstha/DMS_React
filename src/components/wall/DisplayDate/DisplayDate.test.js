import React from 'react';
import DisplayDate from './index';

import testSnapFunction from '../../../util/testSnapFunction';

testSnapFunction("<DisplayDate/>", "Snapshot test for PhroductGridItem",<DisplayDate /> )