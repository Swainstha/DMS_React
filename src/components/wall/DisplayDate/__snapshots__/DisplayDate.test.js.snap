// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DisplayDate/> Snapshot test for PhroductGridItem 1`] = `ShallowWrapper {}`;
